android/app/src/bte/java/covidsafepaths/bte/MainApplication.java

```diff
@@ -0,0 +1,81 @@
+package covidsafepaths.bte;
+
+import android.app.Application;
+import android.content.Context;
+import com.facebook.react.PackageList;
+import com.facebook.react.ReactApplication;
+import com.facebook.react.ReactNativeHost;
+import com.facebook.react.ReactPackage;
+import com.facebook.soloader.SoLoader;
+import java.lang.reflect.InvocationTargetException;
+import java.util.List;
+import org.pathcheck.covidsafepaths.BuildConfig;
+
+public class MainApplication extends Application implements ReactApplication {
+
+  private static Context context;
+
+  private final ReactNativeHost mReactNativeHost =
+      new ReactNativeHost(this) {
+        @Override
+        public boolean getUseDeveloperSupport() {
+          return BuildConfig.DEBUG;
+        }
+
+        @Override
+        protected List<ReactPackage> getPackages() {
+          @SuppressWarnings("UnnecessaryLocalVariable")
+          List<ReactPackage> packages = new PackageList(this).getPackages();
+
+          return packages;
+        }
+
+        @Override
+        protected String getJSMainModuleName() {
+          return "index";
+        }
+      };
+
+  @Override
+  public ReactNativeHost getReactNativeHost() {
+    return mReactNativeHost;
+  }
+
+  @Override
+  public void onCreate() {
+    super.onCreate();
+    MainApplication.context = getApplicationContext();
+    SoLoader.init(this, /* native exopackage */ false);
+    initializeFlipper(this); // Remove this line if you don't want Flipper enabled
```

Comment:
Do we want Flipper here? Im only vaguely aware of it as a debugging tool for RN? not sure if it something we are planning on using and its probably fine to leave in either way, but wondering if you have any context on what we are going to be using it for.

Reply:
Good question - I'd err on the side of leaving it in, assuming it's being used by other contributors.